src/Native/Silk.NET.Assimp.Native/Silk.NET.Assimp.Native.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <SilkNativePackage>true</SilkNativePackage>
-  </PropertyGroup>
```

Comment:
Why? This makes it exempt from SourceLink and common stuff we probably don't want. I might be missing something. Applies to all.

Reply:
`Directory.Build.props` sets `SilkNativePackage` which in turn causes the properties I removed to be set in `common.props`.

Reply:
Nice! I wasn’t aware there was duplication, will resolve these.